### ISSUE TITLE
Add default split points for reserved sections of the metadata table

### ISSUE THREAD

**Issue by ddanielr** [apache/accumulo#4519]
**Is your feature request related to a problem? Please describe.**
As part of the init process, accumulo should add split points to the metadata table for the different sections (sserv, del, prob,blip, etc) so that they are initially contained within their own tablet files. 

This will help with data organization.

**Comment by ctubbsii** [comment]
I don't think we need to do this for 2.1. It may be useful in future versions, but we should be aware that it's possible other best practices (deleting empty splits in the metadata) may conflict with this change, if those best practices are also used, and one isn't careful about when/how to use them together with this change. We could make it so that specific split points cannot be deleted at all... but I'm not sure that's a good idea either. If all these reserved sections are moved to their own separate tables in the accumulo namespace, that would avoid the need to do this.

I seem to recall we also recently discussed the idea of possibly moving the `~del` section to be interspersed throughout the tablets, but I've momentarily forgotten why that was proposed, and don't have time to look up the ticket at the moment.
